cmd/dymd/cmd/inspect_tendermint.go

```diff
@@ -7,10 +7,10 @@ import (
 
 	dbm "github.com/cometbft/cometbft-db"
 
-	cfg "github.com/tendermint/tendermint/config"
-	"github.com/tendermint/tendermint/libs/os"
-	"github.com/tendermint/tendermint/state"
-	"github.com/tendermint/tendermint/store"
+	cfg "github.com/cometbft/cometbft/config"
```

Comment:
if this is now all about cometbft, then the file and relevant parts could also be renamed accordingly